src/integrationtests/java/com/aws/greengrass/integrationtests/e2e/BaseE2ETestCase.java

```diff
@@ -419,7 +428,8 @@ protected void setupTesRoleAndAlias() throws InterruptedException {
         deviceProvisioningHelper.setupIoTRoleForTes(TES_ROLE_NAME, TES_ROLE_ALIAS_NAME, thingInfo.getCertificateArn());
         if (tesRolePolicyArn == null || !tesRolePolicyArn.isPresent()) {
             tesRolePolicyArn = deviceProvisioningHelper
-                    .createAndAttachRolePolicy(TES_ROLE_NAME, TES_ROLE_POLICY_NAME, TES_ROLE_POLICY_DOCUMENT);
```

Comment:
Just realized this, should we change e2e tests to target prod?
Maybe add a separate task for this.

Reply:
Not sure if we want to, should discuss